### ISSUE TITLE
advice for RStudio users?

### ISSUE THREAD

**Issue by jrosen48** [jooyoungseo/jladown#2]
The (helpful) example in the [README](README.md) suggests that users should do the following if they do *not* use RStudio:

```{r}
## Do the following if you are a non-RStudio user:

rmarkdown::draft(file = "my_article", 
   template = "jla_article", 
   package = "jladown"
)
```

What should RStudio users do?

**Comment by jooyoungseo** [comment]
@jrosen48, this is a part for which I need your help. At this moment, I cannot check this aspect since RStudio Desktop has not yet been accessible as of this writing. 

Please read [1.4.3 Creating a document](https://crsh.github.io/papaja_man/introduction.html#creating-a-document) as described in `papaja` package document to get a sense of how Rmd template works because the procedure would be the same. We will detail RStudio's walk-through in our document and README as well.

In short, please see if you could find a template by clicking RStudio's *Knit* button after loading `jladown` package in your RStudio session:

``` r
library(jladown)
```

Once you find the workaround for RStudio's users, please commit a PR after updating the `README.md` file.

**Comment by jrosen48** [comment]
Doing this!

**Comment by jooyoungseo** [comment]
I will make a [project template skeleton](https://rstudio.github.io/rstudio-extensions/rstudio_project_templates.html) for RStudio users so that they can easily draft their manuscripts.

Stay tuned! I am working on it.

**Comment by jooyoungseo** [comment]
Made a basic project template for RStudio users via 210937a.

It requires @jrosen48's test.

**Comment by jooyoungseo** [comment]
README has been updated via b832301. But, please push a new PR for README after testing this feature.

**Comment by jrosen48** [comment]
Two quick questions (I'm sorry for delay). 

1. Should the command be `jladown::jladown_skeleton('.')` (with two colons) or `jladown:::jladown_skeleton('.')` (with three colons)? I was expecting to use the first of the two commands (with two colons) - and think that would be my preference (because the two would indicate that the function is exported), but would love to hear your thoughts here.

2. While the command with two colons work, there does not appear to be any documentation for the function. Should I/we add this, too?

**Comment by jrosen48** [comment]
Edit - removed my third question, sorry, I misread the directions

**Comment by jrosen48** [comment]
Pushed a PR for README after testing this (in #5)

**Comment by jooyoungseo** [comment]
@jrosen48,

Would you do me a favor? Please check if you could also create a jladown project template (this is different from the way you would make like `papaja`) as described in [project template skeleton](https://rstudio.github.io/rstudio-extensions/rstudio_project_templates.html).

I have also implemented this mechanism to `jladown` package and I think this would be better approach than default Rmd template because this can provide a `bookdown-like` skeleton.

If this also works, I would like to walk users through this way rather than Rmd drafting template menus.

**Comment by jrosen48** [comment]
Got it. It does *not yet* appear to be there, but I don't know why.

is the [template metadata](https://rstudio.github.io/rstudio-extensions/rstudio_project_templates.html#defining-the-template-metadata) provided in the package? Hmm...

**Comment by jrosen48** [comment]
I spoke 1 minute too soon - I'm sorry, I had to re-build the package - it is appearing there! Nice job! More in a moment, but just wanted to correct the record.

**Comment by jooyoungseo** [comment]
> Two quick questions (I'm sorry for delay).
> 
> 1. Should the command be `jladown::jladown_skeleton('.')` (with two colons) or `jladown:::jladown_skeleton('.')` (with three colons)? I was expecting to use the first of the two commands (with two colons) - and think that would be my preference (because the two would indicate that the function is exported), but would love to hear your thoughts here.
> 2. While the command with two colons work, there does not appear to be any documentation for the function. Should I/we add this, too?

@jrosen48,

`jladown:::jladown_skeleton` will be documented later so that it can be imported using two colons (`::`). At this stage, this function is internally utilized for the project template capability, which is still under development. I will ask for your review once it gets done.

**Comment by jrosen48** [comment]
makes sense, sounds good, thank ya!
